package.json

```diff
@@ -252,34 +252,34 @@
     "watch": "tsc -watch -p ./"
   },
   "devDependencies": {
-    "@types/mocha": "^5.2.6",
-    "@types/node": "11.13.8",
-    "typescript": "^3.4.5",
-    "vscode": "^1.1.33"
+    "@types/mocha": "^5.2.7",
+    "@types/node": "12.7.2",
+    "typescript": "^3.5.3",
+    "vscode": "^1.1.36"
   },
   "dependencies": {
     "@types/ssh2": "^0.5.38",
     "azure-arm-resource": "^7.3.0",
     "azure-storage": "^2.10.3",
-    "fs-extra": "^7.0.1",
+    "fs-extra": "^8.1.0",
     "fuzzaldrin-plus": "^0.6.0",
     "git-clone": "^0.1.0",
-    "ms-rest": "^2.5.0",
-    "ms-rest-azure": "^2.6.0",
+    "ms-rest": "^2.5.3",
+    "ms-rest-azure": "^2.5.5",
```

Comment:
I just want to confirm that we intend to decrease the version of ms-rest-azure. Right?

Reply:
yes, it's caused by an issue in azure-arm-resource package. azure-arm-resource 7.3.0 depends on ms-rest-azure 2.5.5, not 2.6.x.

Reply:
@MyronFanQiu fixed, could you pls review again